torch_frame/data/mapper.py

```diff
@@ -252,3 +252,60 @@ def forward(
 
     def backward(self, tensor: Tensor) -> pd.Series:
         raise NotImplementedError
+
+
+class TextTokenizationTensorMapper(TensorMapper):
+    r"""Tokenize any text series into a dictionary of
+    :class:`MultiNestedTensor`.
+
+    Args:
+        text_tokenizer (callable): A callable function that takes list of
+            strings and returns a dictionary of :class:`MultiNestedTensor`.
+            The keys of the dictionary are arguments that will be put to the
+            model, including :obj:`input_ids` and :obj:`attention_mask`.
+            The values of the dictionary are tensors corresponding to keys.
+        batch_size (int, optional): The mini-batch size used for the text
+            tokenizer. If :obj:`None`, we will encode all text in a full-batch
+            manner.
+    """
+    def __init__(
+        self,
+        text_tokenizer: Callable[[List[str]], Dict[str, MultiNestedTensor]],
```

Comment:
I think `text_tokenizer` here should be drop-in replacement of existing tokenizer, so we should avoid the dependency on `MultiNestedTensor`